_posts/2018-08-31-grpc-packaging-releases-nightlies.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: gRPC Packaging: Release Process and Nightly Builds
+published: true
+permalink: blog/grpc-packaging-releases-nightlies
+author: Mehrdad Afshari
+author-link: https://mehrdad.afshari.me
+company: Google
+company-link: https://www.google.com
+---
+
+# gRPC Packaging: Release Process and Nightly Builds #
+
+gRPC is [developed in the open on GitHub](https://github.com/grpc) and the
+`master` branch undergoes continuous testing to try to ensure the `HEAD` of
+`master` is buildable and passes the tests.  In this post, we discuss the
+release process of gRPC as an open source project to clarify it for our users,
+and to serve as an example for other open source projects to consider adopting.
```

Comment:
I would avoid stating that we want this to serve as an example for other open source projects. 
"In this post we discuss the release process of gRPC" should be sufficient..

Reply:
Why?